Fix double click to show exercise, double login submission on enter

AbraaoMota opened the pull request:
Removes double click issues when showing an exercise and duplicate message sending on logging in using the Enter key.
Minor visual fix to buttons on annotation editor.

mihaicar commented on infrastructure/microservices.py:
```diff
@@ -5,6 +5,5 @@
 
 microservices = [
     {"name": "linter", "location": "java -jar backend/linter/target/linter-1.0-SNAPSHOT-jar-with-dependencies.jar" },
-    {"name": "cleaner", "location": "./infrastructure/cleaner"},
```
wasnt this supposed to be the comment cleaner microservice?

giuliojiang replied:
it's not the 'real' cleaner, this was just a placeholder that I put there when there were no microservices written. The 'real' cleaner should be hooked in by Elias (I believe he wrote it right?)

AbraaoMota committed:
Fix double click to show exercise, double login submission on enter

Removes double click issues when showing an exercise and duplicate message sending on logging in using the Enter key.
Minor visual fix to buttons on annotation editor.

helicopter88 reviewed (approved):
LGMT

giuliojiang reviewed (approved):
very good

eliasbenussi reviewed (approved):
LGTM
